Enhances command spo list remove (#6378)
pnp/cli-microsoft365: docs/docs/cmd/spo/list/list-remove.mdx

`-t, --title [title]`
: Title of the list to remove. Specify either `id` or `title` but not both.

`--recycle`

> : Recycle the list

Reviewer: This is not the description provided in the GitHub issue.

Author: Updated

m365 spo list remove --webUrl https://contoso.sharepoint.com/sites/project-x --title 'List 1'
```

> Remove the list with a specific ID located in a specific site and send it to recycle bin

Reviewer: ```suggestion
Remove a list specified by id by sending it to the recycle bin instead of permanently removing it
```

Author: Updated
